app/helpers/validation_request_helper.rb

```diff
@@ -38,4 +38,12 @@ def display_request_status(validation_request)
       "green"
     end
   end
+
+  def overdue_requests(validation_requests)
+    validation_requests.select { |req| req.overdue? && req.state == "open" }
+  end
+
+  def closed_requests(validation_requests)
+    validation_requests.select { |req| req.state == "closed" }
+  end
```

Comment:
I think these two methods make more sense on the planning_application model than a helper.